packages/utils/telemetry-utils/src/utils.ts

```diff
@@ -25,8 +21,8 @@ import { ITelemetryGenericEventExt, ITelemetryLoggerExt } from "./telemetryTypes
  */
 export function logIfFalse(
 	condition: unknown,
-	logger: ITelemetryBaseLogger,
-	event: string | ITelemetryGenericEvent,
+	logger: ITelemetryLoggerExt,
+	event: string | ITelemetryGenericEventExt,
```

Comment:
These changes would not be correct because `logIfFalse` is exported by this package, so consumers can use it and they should not be exposed to the `*Ext` types. That said, we deprecated this function in 7.4 so it's a good time to remove it. I'll put out a separate PR to do that, so these changes should disappear from this one.

Reply:
[PR to remove it](https://github.com/microsoft/FluidFramework/pull/19597).

Reply:
Resolved after rebase. Thx!